src/packages/react-force-graph-2d/index.d.ts

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
 import { ForceGraphInstance as ForceGraphKapsuleInstance } from 'force-graph';
 
-export interface GraphData {
-  nodes: NodeObject[];
-  links: LinkObject[];
+export interface GraphData<NodeType = {}, LinkType = {}> {
+  nodes: NodeObjectIntersection<NodeType>[];
+  links: LinkObjectIntersection<NodeType, LinkType>[];
 }
 
-export type NodeObject = object & {
+export type NodeObject = {
```

Comment:
I'm thinking if doing something like this would eliminate the need for the additional `NodeObjectIntersection` type:
```ts
export type NodeObject<NodeType = {}> = NodeType & {
  id?: string | number;
  x?: number;
  y?: number;
  vx?: number;
  vy?: number;
  fx?: number;
  fy?: number;
  [others: string]: any;
}
```

Reply:
I didn't want to modify the original `NodeObject` and `LinkObject` types when adding the intersection types. However, since those types had already been modified in the original commits, they can be modified to remove the new intersection types.

I have applied the changes, as suggested.